examples/modal/app.R

```diff
@@ -0,0 +1,47 @@
+library(shiny)
+library(shiny.semantic)
+library(shinyjs)
+
+ui <- function() {
+  shinyUI(
+    semanticPage(
+      useShinyjs(),
+      title = "Modal example",
+      uiOutput("modalBind"),
+      uiOutput("modalAction"),
+      div(id = "modal-button", class = "ui button", "Im a button binded by the modal target parameter"),
+      actionButton("show", "I am an action button calling show_modal")
+    )
+  )
+}
+
+server <- shinyServer(function(input, output) {
+  observeEvent(input$show, {
+    show_modal('#action-example-modal')
```

Comment:
The idea was to enable users to use any selector for the element. however since the modal ID will always be set makes sense to keep it element ID only.